at_client/lib/src/service/encryption_service.dart

```diff
@@ -119,6 +126,11 @@ class EncryptionService {
   /// Used for local lookup @bob:phone@alice
   Future<String?> decryptLocal(String? encryptedValue, String? currentAtSign,
       String sharedWithUser) async {
+    if (encryptedValue == null || encryptedValue.isEmpty) {
```

Comment:
can we also log in the caller when encryptedValue is null.so we will know for which key the encryptedValue is null